src/components/Hero.tsx

```diff
@@ -35,7 +35,7 @@ const Content = ({
         <SuomiFi
           style={{
             fill: suomifiTheme.colors.whiteBase,
-            fontSize: compact ? '30px' : '63px',
+            fontSize: compact ? '30px' : '64px',
```

Comment:
These could also come from theme object. If not found, rebase develop (and yarn install and yarn develop) and typography tokens should be available.

Reply:
Not sure what have been the used conventions, but I personally like comments with magic numbers and agree with the above comment.

Reply:
I'll look into it soon.

Reply:
It is also fine for now if too much work to do within this fix, as I can see that it might get refactored a bit later on when using more ready made components.